ui/component/or-mwc-components/src/or-mwc-tabs.ts

```diff
@@ -142,7 +143,7 @@ export class OrMwcTabs extends LitElement {
                 <div class="mdc-tab-bar" role="tablist" id="tab-bar">
                     <div class="mdc-tab-scroller">
                         <div class="mdc-tab-scroller__scroll-area">
-                            <div class="mdc-tab-scroller__scroll-content">
+                            <div class="mdc-tab-scroller__scroll-content" @click="${() => { console.log(this.mdcTabBar); }}">
```

Comment:
Can remove this

Reply:
For now left it in there for testing purposes of fixing or-mwc-tabs issue ( #746 )
Will fix of course before merge.